content/quickstart/azure/create-project.md

```diff
@@ -0,0 +1,162 @@
+
+After logging in, the CLI will proceed with walking you through creating a new project.
+
+```
+This command will walk you through creating a new Pulumi project.
+
+Enter a value or leave blank to accept the (default), and press <ENTER>.
+Press ^C at any time to quit.
+
+project name: (quickstart)
+project description: (A minimal Azure Pulumi program)
+Created project 'quickstart'
+
+stack name: (dev)
+Created stack 'dev'
+
+azure:environment: The Azure environment to use (`public`, `usgovernment`, `german`, `china`): (public)
+Saved config
+```
+
+First, you will be asked for a project name and description. You can hit enter to accept the default values or enter a new values.
+
+Next, you will be asked for the name of a stack. You can hit enter to accept the default value of `dev`.
+
+> What are [projects]({{< relref "/reference/project.md" >}}) and [stacks]({{< relref "/reference/stack.md" >}})? Pulumi projects and stacks are a way to organize Pulumi code. You can consider a Pulumi Project to be analogous to a GitHub repo: a single place for code — and a Stack to be an instance of that code which has separate configuration. For instance, Project Foo may have multiple stacks for Dev, Test, Prod, or perhaps for different cloud configurations (e.g. geographic region). Please [see this guide]({{< relref "/reference/organizing-stacks-projects.md" >}}) for some best practices on organizing your Pulumi projects and stacks.
+
+Next, you will be prompted for some configuration values for the stack.
+
+For Azure projects, you will be prompted for the Azure environment. You can accept the default value of `public` or choose another environment.
+
+{{% lang nodejs %}}
+After some dependency installations from `npm`, you'll see some files have been generated:
+{{% /lang %}}
+
+{{% lang python %}}
+After the command completes, you'll see some files have been generated:
+{{% /lang %}}
+
+- `Pulumi.yaml` defines the [project]({{< relref "/reference/project.md" >}}).
+- `Pulumi.dev.yaml` contains [configuration]({{< relref "/reference/config.md" >}}) values for the [stack]({{< relref "/reference/stack.md" >}}) we initialized.
+- {{< langfile >}} is the Pulumi program that defines our stack resources. Let's examine it.
+
+{{< langchoose nogo >}}
+
+```javascript
+"use strict";
+const pulumi = require("@pulumi/pulumi");
+const azure = require("@pulumi/azure");
+
+// Create an Azure Resource Group
+const resourceGroup = new azure.core.ResourceGroup("resourceGroup", {
+    location: "WestUS",
+});
+
+// Create an Azure resource (Storage Account)
+const account = new azure.storage.Account("storage", {
+    resourceGroupName: resourceGroup.name,
+    location: resourceGroup.location,
```

Comment:
This isn't necessary, since we automatically flow the location now.

Reply:
Oh yeah, I forgot about your improvement! I've left this as-is for now, as this is the current output of the `pulumi new azure-typescript` template (and others). I'll fix-up the template and update it in the Get Started guide subsequently.